aopy/postproc/base.py

```diff
@@ -384,3 +385,38 @@ def get_calibrated_eye_data(eye_data, coefficients):
     """    
     #caliberated_eye_data_segments = np.empty((num_time_points, num_dims))
     return eye_data * coefficients[:,0] + coefficients[:,1]
+
+def smooth_timeseries_gaus(timeseries_data, sd, samplerate, nstd=3, conv_mode='same'):
+    """
+    Smooths a time series by convolving it with a user-defined Gaussian kernel. The Gaussian kernel is defined by the following equation:
+
```

Comment:
Describing how filter length (corresponding x in this function) is calculated may be good for users.